src/context.rs

```diff
@@ -622,7 +622,7 @@ impl Context {
 
     pub fn get_font_face(&self) -> FontFace {
         unsafe {
-            FontFace(ffi::cairo_get_font_face(self.0))
+            from_glib_none(ffi::cairo_get_font_face(self.0))
```

Comment:
Yes it was bug, 
https://www.cairographics.org/manual/cairo-text.html#cairo-get-font-face cleary says that reference needed.